README.md

```diff
@@ -37,7 +37,8 @@ The following provides a non-comprehensive overview:
 
 - 🛠️ Custom parameter encodings: Improve your campaign with domain knowledge
 - 🧪 Built-in chemical encodings: Improve your campaign with chemical knowledge
-- 🎯 Single and multiple targets with min, max and match objectives
+- 🎯 Numerical and binary targets with min, max and match objectives

```

Comment:
@Scienfitz: Had to drop the `multiple` from this line in order to make room for the new Pareto feature. Feel free to adjust/improve